tests/test_ctl.py

```diff
@@ -4,6 +4,8 @@
 
 import pytest
 
+from psycopg2 import connect
```

Comment:
you can import this locally in `test_func_start_stop_status_restart_reload` to work around CI error on job "deb-buster"